elements/virtual-waggle/post-install.d/02-virtual-waggle

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+if [ \${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+mkdir -p /opt/virtual-waggle
+mkdir -p /opt/virtal-waggle/private
+
+cd /opt/virtual-waggle
+
+git clone https://github.com/waggle-sensor/waggle-node.git .
+
+rm .env
+
+echo "VW_ROOT=/opt/virtual-waggle" >> /etc/environment
+echo "WAGGLE_ETC_ROOT=/opt/virtual-waggle/private" >> /etc/environment
+
+cat > /usr/local/sbin/virtual-waggle  <<EOF
```

Comment:
If I'm reading this right, it looks like this is duplicating a lot of the `waggle-node` script, right?

If we really do want to provide a "global" virtual-waggle command, I'd strongly recommend instead just doing an alias like:

```sh
#!/bin/bash

cd /opt/virtual-waggle/waggle-node
./waggle-node $*
```

Reply:
Wow, that's way easier. I wish I thought about that. I don't work with shell scripts enough.

Reply:
It's cool - just wanted to make your job easier. :)